doc/source/examples/similarity_explanations_mnist.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<p>I would replace '<code>============='</code>by  <code>len(title) * '='</code> to make sure that they cover the entire title.</p><p> </p><p>Probably consider adding documentation and a brief description of the parameters.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>